docs/Views/main.js

```diff
@@ -0,0 +1,36 @@
+import loginFunctions from '../firebase/login.js';
+
+export default () => {
+  const loginView = document.createElement('div');
+  loginView.setAttribute('class', 'viewsStyle');
+  loginView.innerHTML = ` 
+            <center><h3>Inicia Sesión con tu email</h3></center>
+            <form class="registerForm">
+                <input type="email" required id="loginEmail" placeholder= "Correo electrónico">
+                <input type="password" required id="loginPassword" minlength="6" placeholder="Contraseña">
+                <button type="submit" class="registerButton" id="loginButton">INICIAR SESIÓN</button>
+                <h5><a href="#/passwordRecovery" id="recover">¿Has olvidado la contraseña?</a></h5>
+            </form> 
+            <div class="conectWithSocialMedia">
```

Comment:
si la clase definida es para implementar estilos, no se recomienda el uso de camel case
```suggestion
            <div class="conect-with-social-media">
```